sim_pipeline/sim_pipeline.py

```diff
@@ -1 +1,23 @@
 """Main module."""
+
+import numpy as np
+import copy
+import matplotlib.pyplot as plt
+
+from .transient_generation import LensedTransientImage,LensedTransientSystem
+
+
+def LensedSystem_from_LensParams(lensed_transients, time_delays, magnifications):
```

Comment:
can you document the inputs to this definition? Also, definitions are lower_case_wordings

Reply:
This function is sort of temporary, pending the connection point to the lens sim part so I'll leave it for now